feature/stats/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.track.stats"
+    compileSdk = 33
+
+    defaultConfig {
+        minSdk = 24
+        targetSdk = 33
```

Comment:
phần này sử dụng biến chung trong buildSrc đc ko e? Mấy giá trị này thường khi thay đổi thì thay đổi toàn app. Nếu mình để riêng thì sửa hơi rắc rối